tests/foreman/ui/test_computeprofiles.py

```diff
@@ -42,13 +42,22 @@ def test_positive_end_to_end(session, module_loc, module_org):
     ).create()
     with session:
         session.computeprofile.create({'name': name})
-        assert session.computeprofile.search(name)[0]['Name'] == name
+        assert (
+            entities.ComputeProfile().search(query={'search': 'name={0}'.format(name)}),
+            'Compute profile {0} expected to exist, but is not included in the search results'.format(name)
+        )
```

Comment:
I'm afraid Python understands this as "given this two-value tuple, where first value is function call result, and second value is formatted string, assert that tuple is true-ish", which it always will be.

You want something like below instead:
```python
assert entities.ComputeProfile().search(query={'search': 'name={0}'.format(name)}), (
    'Compute profile {0} expected to exist, but is not included in the search results'.format(name))
```

Reply:
yeah, i already fixed this one, now i'm hitting other issues, added `do not merge` label until i manage to fix it up